### PR TITLE
Clean up RPM specification

### PR DESCRIPTION
**Pull request by aimylios**
This is a collection of (mostly cosmetic) changes I made to clean up the SPEC file.

Next on my list is reintegration of the doc package, but I'll save that for another day.

### CODE REVIEW

File: kicad.spec.template [hunk 1]
```diff
@@ -180,6 +181,10 @@ popd
 %find_lang %{name}
 
 
+%check
+appstream-util validate-relax --nonet %{buildroot}/%{_datadir}/appdata/*.appdata.xml
```
Comment (nickoe) [inline kicad.spec.template:185]: Just for my learning, what is this appstream appdata stuff for?

**Review by nickoe — APPROVED** [review]
Thank your for submitting small and atomic patches with good descriptions. It makes it a breeze to review. I am not too familiar with the spec syntax, so I did not review those in detail but overall the changes look good. I am not sure about that appdata stuff, but I will merge anyway.

**Comment by aimylios** [comment]
Thank you for merging! I see there was a [failed build](https://copr.fedorainfracloud.org/coprs/g/kicad/kicad/build/721956/) last night, I hope that was not related to my changes. At least I didn't encounter any issues with my local build.

The AppStream data is used by Linux application installers like KDE Discover to make the installation of new applications more user-friendly (similar to an "App Store"). Fedora started shipping AppData files for KiCad in April 2016 as a patch [[1]](https://src.fedoraproject.org/rpms/kicad/c/b9471970a5033f77b7e3828ef304140a734f70d3?branch=master) [[2]](https://src.fedoraproject.org/rpms/kicad/c/6390d9c37f42d326f4dad9b1dff40fd75480826a?branch=master), and it was later officially included into the KiCad sources [[3]](https://github.com/KiCad/kicad-source-mirror/commit/862711b71a78b140ee2987eeccb44a29a4f2a14f).

The call to `appstream-util` I backported just checks whether the AppData that was installed complies to the AppStream specification. As the installation process does not actually touch these files but just copies them, it would probably be sufficient to do this only once when modifying the files (which didn't happen up to now, they are still identical to what was submitted a year ago), and not after every (nightly) build.

I merely included this in an attempt to keep the nightly SPEC as much as possible in sync with what stable packaging does. And I deliberately put this patch at the end of the PR to make it easier to remove in case someone complains. :-)
So please feel free to revert it if you think that it's not worth to complicate the build process just to have some more code in common with upstream.

**Comment by nickoe** [comment]
I think it is fine to keep

**Pull request merged**